File: app/src/androidTest/java/com/fractalwrench/crazycats/AppSmokeTest.java
package com.fractalwrench.crazycats;

import android.content.Context;
import android.support.test.InstrumentationRegistry;
import android.support.test.runner.AndroidJUnit4;

import org.junit.Test;
import org.junit.runner.RunWith;

import static org.junit.Assert.assertTrue;

/**
 * Ensures that the application builds and launches on an Android device.
 */
@RunWith(AndroidJUnit4.class)
public class AppSmokeTest {

    @Test
    public void useAppContext() throws Exception {
        Context appContext = InstrumentationRegistry.getTargetContext();
        assertTrue(appContext.getPackageName().startsWith("com.fractalwrench.crazycats"));
    }

}

File: app/src/androidTest/java/com/fractalwrench/crazycats/AppSmokeTest.kt
package com.fractalwrench.crazycats

import android.support.test.InstrumentationRegistry
import android.support.test.runner.AndroidJUnit4
import org.junit.Assert.assertTrue
import org.junit.Test
import org.junit.runner.RunWith

/**
 * Ensures that the application builds and launches on an Android device.
 */
@RunWith(AndroidJUnit4::class)
class AppSmokeTest {

    @Test
    @Throws(Exception::class)
    fun useAppContext() {
        val appContext = InstrumentationRegistry.getTargetContext()
        assertTrue(appContext.packageName.startsWith("com.fractalwrench.crazycats"))
    }

}
